pyomo/solvers/tests/models/LP_duals_minimize.py

```diff
@@ -8,8 +8,8 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-import pyomo.kernel as pmo
```

Comment:
I think we should keep this import as `pmo` to avoid confusion between AML and kernel components